components/channel_header/channel_header.js

```diff
@@ -105,6 +105,8 @@ class ChannelHeader extends React.PureComponent {
         this.getPopoverMarkdownOptions = memoizeResult((channelNamesMap) => (
             {...popoverMarkdownOptions, channelNamesMap}
         ));
+
+        this.headerOverlay = createRef();
```

Comment:
Can we please group this with the other refs declared at the top of the constructor (ln. 96-98), include 'Ref' at the end of the variable name to match (i.e. `headerOverlayRef`) and settle on either using `React.createRef()` or `createRef()` directly for consistency? Might be best to just use `React.createRef()` in this case as that's how the rest of the file is using React methods.